components/Msig/Create/Confirm.jsx

```diff
@@ -8,21 +8,23 @@ import {
   Box,
   IconPending,
   StyledATag,
+  Button,
   Text,
+  Glyph,
   Title,
   BigTitle
 } from '../../Shared'
 import { fetchAndSetMsigActor } from '../../../store/actions'
 
 const NextOption = ({ text, onClick }) => {
   return (
-    <Card
+    <Button
+    title={text}
```

Comment:
If your tests fail, it's probably because this line throws a linter error?

`npm run lint` to find out